script.kodi.windows.update/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.kodi.windows.update" name="Kodi Windows Installer" version="1.3.2" provider-name="Team Kodi">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.six" version="1.0.0"/>
+        <import addon="script.module.kodi-six" version="0.1.3.1"/>
+        <import addon="script.module.simplecache" version="1.0.0"/>
+        <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+    </requires>
+    <extension point="xbmc.python.script" library="default.py"/>
+    <extension point="xbmc.service" library="service.py"/>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Download and Install Kodi packages.</summary>
+        <description lang="en_GB">Download and Install Kodi window executables; select from  Nightlies, Releases, Snapshots and Test-builds.</description>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
+        <platform>windows</platform>
+        <website>http://kodi.tv</website>
+        <source>https://github.com/Lunatixz/KODI_Addons/tree/master/script.kodi.windows.update</source>
+        <forum>https://forum.kodi.tv/showthread.php?tid=325226</forum>
+        <disclaimer lang="en_GB">UWP not supported! Not intended for Kodi installation through the Windows Store.</disclaimer>
```

Comment:
can't this be filtered in the platform tag? I updated the values a while ago based on the code: https://kodi.wiki/view/Addon.xml#.3Cplatform.3E

Reply:
Those new? I guess 'windows-x86_64' would apply here. I'll make the change.

Reply:
Better to test, I don't have a windows machine. Those are not new, just not documented :)

Reply:
I think what you're looking for is windx. This means windows without uwp (windowsstore). But yeah those values need to be updated on the checker